File: src/test/java/pw/aru/javabot/AboutCommand.java
package pw.aru.javabot;

import net.dv8tion.jda.api.EmbedBuilder;
import org.jetbrains.annotations.NotNull;
import pw.aru.psi.commands.Category;
import pw.aru.psi.commands.Command;
import pw.aru.psi.commands.ICategory;
import pw.aru.psi.commands.context.CommandContext;

import static pw.aru.psi.exported.PsiExported.psi_version;

@Command({"about", "thanks"})
@Category("j#info")
class AboutCommand extends AbstractCommand {
    public AboutCommand(ICategory c) {
        category = c;
    }

    @Override
    public void call(@NotNull CommandContext ctx) {
        ctx.getChannel().sendMessage(
            new EmbedBuilder()
                .appendDescription("This is an example Java bot made using psi " + psi_version + ".\nThanks for using it <3.")
                .build()
        ).submit();
    }
}

File: src/test/kotlin/pw/aru/testbot/AboutCommand.kt
package pw.aru.testbot

import pw.aru.psi.commands.Category
import pw.aru.psi.commands.Command
import pw.aru.psi.commands.ICategory
import pw.aru.psi.commands.ICommand
import pw.aru.psi.commands.context.CommandContext

@Command("about", "thanks")
@Category("info")
class AboutCommand(override val category: ICategory) : ICommand {
    override fun CommandContext.call() {
        sendEmbed {
            description("<3 for using our bot.")
        }
    }
}
